templates/login/login_core.html.twig

```diff
@@ -1,14 +1,16 @@
-{% set srOnly = (showLabels == true) ? "" : "sr-only" %}
 <!DOCTYPE html>
 <html lang="en">
 <head>
-    {% if displayGoogleSignin %}
-        <meta name="google-signin-client_id" content="{{ googleSigninClientID|attr }}">
-    {% endif %}
+   <script src="https://accounts.google.com/gsi/client" async defer></script>
+    <div id="g_id_onload"
+         data-client_id="{{ googleSigninClientID }}"
```

Comment:
hi @Saparbek-Nagashibekov, thanks for the PR, think this needs to stay as `googleSigninClientID|attr`